src/Generator.Bind/CSharpSpecWriter.cs

```diff
@@ -410,7 +410,29 @@ private void WriteConstants(BindStreamWriter sw, IEnumerable<Constant> constants
                 if (!Settings.IsEnabled(Settings.Legacy.NoDocumentation))
                 {
                     sw.WriteLine("/// <summary>");
-                    sw.WriteLine("/// Original was " + Settings.ConstantPrefix + c.OriginalName + " = " + c.Value);
+
+                    string requires_string = "";
+                    if (string.IsNullOrEmpty(c.AdddedInExtension) == false)
+                    {
+                        requires_string = "[requires: ";
+
+                        // This constant was added in an extension
+                        if (c.AdddedInExtension.StartsWith("VERSION") && c.AdddedInExtension.Length == "VERSION_X_X".Length)
```

Comment:
I had to read this several times to figure out what I was looking at.  Maybe consider a regex with named captures here instead, and a ternary to reduce nesting:
```cs
private static readonly Regex _versionRegex = new Regex(@"^VERSION_(?<Major>\d+)_(?<Minor>\d+)$");

...

string version = ((match = _versionRegex.Match(c.AdddedInExtension)).Success)
    ? $"v{match.Groups("Major")}.{match.Groups("Minor")}"
    : c.AdddedInExtension;

requires_string = $"[requires: {version}]";
```
This is also more flexible than the original; it's far easier to tweak the regex to describe the pattern than to modify `StartsWith()` and length comparisons and character hacking.

Also:  Wouldn't the correct name be `requiresString`?